docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md

```diff
@@ -71,7 +71,7 @@ If the DNS record exists it returns the answer and sets query status to `NO_ERRO
 
 #### System Requirements for Mode `run`
 
-`ziti-edge-tunnel run` requires elevated privileges for managing the `/dev/net/tun` device, running `resolvectl` to assign nameservers and domain routes to the tun interface, and running `ip route` to manage IP routes. This requires running as root because `setcaps` are not inherited in all of these cases, even when the inherit bit is set.
+`ziti-edge-tunnel run` requires elevated privileges for managing the `/dev/net/tun` device, running `resolvectl` to assign nameservers and domain routes to the tun interface, and running `ip route` to manage IP routes. This is normally accomplished by PolKit rules defined in the DEB and RPM packages that set capabilities inherited by the run-as user, `ziti`.
```

Comment:
there are actually two kinds of permissions that are required. Each is configured differently:

- tun and route management are permitted by passing the `CAP_NET_ADMIN` to the ziti-edge-tunnel process via `AmbientCapabilities` in the systemd service unit.
- systemd-resolved DNS configuration is permitted by PolKit.